### PR TITLE
Hw 12

### PR DESCRIPTION
**Pull request by centora**

### CODE REVIEW

File: server-app/src/css/app.scss [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
     margin: 0;
-    background: #e4e1c7;
     min-height: 100vh;
     display: flex;
     flex-direction: column;
-    .main {
-        flex: 1;
-        background: $aaa;
-    }
+  	background: #{getDarken(#edecf1)};
```
Comment (iliatcymbal) [inline server-app/src/css/app.scss]: you don't need here the string interpolation

File: server-app/src/scripts/main/main.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+    <li class="item"></li>
+  </ul>
```
Comment (iliatcymbal) [inline server-app/src/scripts/main/main.js:41]: looks ugly... why do not use js to create such strings?

**Review by iliatcymbal — APPROVED** [review]
please pay attention on my comments (for the future time)

File: server-app/src/scripts/main/main.js (hunk 2 above)
Comment (centora) [reply server-app/src/scripts/main/main.js:41]: ) I thought this task connected only to scss , I even didn't notice that I need to use js. I will fix it.

**Pull request closed without merging**